doc/processes/release_process.md

```diff
@@ -32,49 +32,57 @@ and one has an external test repo:
 * https://github.com/eclipse-openj9/openj9-systemtest
 
 OpenJ9 needs to be combined with code from OpenJDK to create a Java SDK.
-The OpenJDK code is managed in separate github repos referred to as the 
-"Extensions for OpenJDKx for OpenJ9".  Currently there are two 
+The OpenJDK code is managed in separate github repos referred to as the
+"Extensions for OpenJDKx for OpenJ9".  There are multiple
 extensions repos:
 
 * JDK8 https://github.com/ibmruntimes/openj9-openjdk-jdk8
 * JDK11 https://github.com/ibmruntimes/openj9-openjdk-jdk11
-* JDK14 https://github.com/ibmruntimes/openj9-openjdk-jdk14
+* JDK17 https://github.com/ibmruntimes/openj9-openjdk-jdk17
+* etc.
 * JDK.next https://github.com/ibmruntimes/openj9-openjdk-jdk
 
 An OpenJ9 release should:
 
 * Use a single code base to support the various JDK-levels.
-* Track the upstream OpenJDK releases.  
-* Use a consistent set of tags across OpenJ9 & extensions repos to 
+* Track the upstream OpenJDK releases.
+* Use a consistent set of tags across OpenJ9 & extensions repos to
 ensure that releases can be rebuilt or branched later.
-* OpenJ9 releases should be high quality and pass a common quality bar. 
-* Communicate the tags for a release to downstream projects. 
+* OpenJ9 releases should be high quality and pass a common quality bar.
+* Communicate the tags for a release to downstream projects.
 * Clearly define the supported platforms
 * Should not regress performance from release to release.
 * Use Github releases to identify releases and link to the relevant
 data.
-* Should tag the openj9-systemtest repo as well so the systemtest level used to validate a build is know.
+* Should tag the openj9-systemtest repo as well so the systemtest level used to validate a build is known.
 
 
 ## Release cadence
-OpenJ9 should aim to release quarterly to match the OpenJDK long term 
-support (LTS) release cadence.  OpenJDK 8 is released quarterly and 
-other LTS releases will likely follow similar release cadences.  There 
-should be monthly milestone builds as the project builds towards the 
+OpenJ9 should aim to release quarterly to match the OpenJDK long term
+support (LTS) release cadence.  OpenJDK 8, 11, 17 are released quarterly and
+other LTS releases will likely follow similar release cadences.  There
+should be milestone builds created as the project advances towards the
 next release.
 
+OpenJ9 will endeavor to finalize a source code release within two weeks of
```

Comment:
```suggestion
OpenJ9 will endeavour to finalize a source code release within two weeks of
```

Reply:
Updated. Pls merge the PR is all is good now.